### ISSUE TITLE
Slideover horizontal scrollbar

### ISSUE THREAD

**Issue by Blixary** [protonemedia/laravel-splade#350]
- Laravel Version: Laravel v10.3.3
- PHP Version: PHP v8.1.6
- Splade JS Version (npm): 1.2.15
- Splade PHP Version (composer): Splade Breeze 2.4 / Composer package version 1.40
- Dev environment (OS, Sail/Valet/etc): Windows, Firefox

### Description:

Opening the slideover component form right to left leads to a vertical scrollbar inside the screen. The scrollbar disappears, when the modal is completely opened. There is no scrollbar when using the slideover from left to right or the modal component.

### Steps To Reproduce Issue:

in modal file:
```
<x-app-layout>
    <x-slot name="header">
        <h2 class="font-semibold text-xl text-gray-800 leading-tight">
            {{ __('Create user') }}
        </h2>
    </x-slot>

<x-splade-modal position="right" max-width="xl">
    Test text

</x-splade-modal>
</x-app-layout>
```

and index file:
`<Link slideover href="/medium/create">Create new user</Link>`

**Comment by pascalbaljet** [comment]
Will be fixed in the next release

**Comment by pascalbaljet** [comment]
Fixed in v1.4.7, make sure to [update both packages](https://splade.dev/docs/upgrading) (JS + PHP).
